pkg/admission/init.go

```diff
@@ -45,6 +48,10 @@ func (i *pluginInitializer) Initialize(plugins []Interface) {
 		if wantsInformerFactory, ok := plugin.(WantsInformerFactory); ok {
 			wantsInformerFactory.SetInformerFactory(i.informers)
 		}
+
+		if wantsAuthorizer, ok := plugin.(WantsAuthorizer); ok {
+			wantsAuthorizer.SetAuthorizer(i.authorizer)
```

Comment:
This is the only use of `WantsAuthorizer` I can find, and it only uses `SetAuthorizer`. So it seems that the `Validator` interface doesn't need to be in this interface.